docker/dev/tls/docker-compose.yml

```diff
@@ -28,10 +28,6 @@ services:
     depends_on:
       step:
         condition: service_healthy
-    logging:
-      options:
-        max-size: "8m"
-        max-file: "10"
```

Comment:
Can you specify the errors thrown by the logging options? @apfohl has a valid point, when he states, he introduced them to reduce produced logs and reduce occupied space on disk.

Reply:
```
Error response from daemon: unknown log opt 'max-file' for journald log driver
```

Same fot `max-size` of course. I guess this is some version incompatibility somewhere. I can see why you'd do this, but it might be better to have it in the overrides then, especially because we don't enable this for every service.